src/main/java/com/whizzosoftware/wzwave/commandclass/VersionCommandClass.java

```diff
@@ -61,20 +61,21 @@ public void onApplicationCommand(NodeContext context, byte[] ccb, int startIndex
         if (ccb[startIndex+1] == VERSION_REPORT) {
             int start = startIndex+2;
             library = String.format("%d", ccb[start]);
-            protocol = String.format("%d.%2d", ccb[start + 1], ccb[start + 2]);
-            application = String.format("%d.%2d", ccb[start + 3], ccb[start + 4]);
+            protocol = String.format("%d.%02d", ccb[start + 1], ccb[start + 2]);
```

Comment:
Programmers guide suggest padding with zeroes ('0'), so added this here. Let me know your thoughts on this.